gulp/tasks/assets.js

```diff
@@ -10,14 +10,22 @@ module.exports = function (gulp, $, config) {
   var faviconsDest = config.paths.favicons.dest;
   var fontsSrc     = config.appFiles.fonts;
   var fontsDest    = config.paths.fonts.dest;
+  var manifestFile = config.paths.revManifest.dest;
 
   var task =  function () {
     var imagesSt = gulp.src(imagesSrc)
       .pipe($.changed(imagesDest))
-      .pipe(gulp.dest(imagesDest));
+      .pipe($.if(config.isProd, $.rev()))
+      .pipe($.if(config.isProd, gulp.dest(imagesDest)))
+      .pipe($.if(config.isProd, $.rev.manifest(manifestFile, { merge: true, base: imagesDest })))
+      .pipe(gulp.dest(imagesDest))
```

Comment:
I don't think so. Here are the two possible situations:

# dev

1. grab source files (first `gulp.src`)
2. do some manipulations
3. save them in build (last `gulp.dest`).

# prod
1. grab source files (first `gulp.src`)
2. do some manipulations
3. rev file
4. save rev'ed file (first `gulp.dest` within an if)
5. generate manifest file
6. save generated manifest file (last `gulp.dest`).